codice/ddf
# DDF-04842 Migrate menu.tsx to styled and parameterize dropdown.js anchor component

#### What does this PR do?
Migrate menu.tsx to styled and parameterize dropdown.js anchor component
#### Who is reviewing it? 

@zta6 @middlets719 @samuelechu 
#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.

@bdeining 
@djblue

#### How should this be tested?
Open up the query menu - select advanced query and select anyGeo.
The Location Dropdown uses this component. Ensure it renders normally, and that changing the value between location types is handled correctly.


#### Any background context you want to provide?
#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #4842

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
